docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -367,12 +367,24 @@ statements, the connector supports the following features:
 - {doc}`/sql/create-table`, see also {ref}`delta-lake-sql-basic-usage`
 - {doc}`/sql/create-table-as`
 - {doc}`/sql/drop-table`
-- {doc}`/sql/alter-table`
+- {doc}`/sql/alter-table`, see also {ref}`delta-lake-alter-table`
 - {doc}`/sql/drop-schema`
 - {doc}`/sql/show-create-schema`
 - {doc}`/sql/show-create-table`
 - {doc}`/sql/comment`
 
+
+(delta-lake-alter-table)=
+
+### ALTER TABLE statement limitations
```

Comment:
Remove "limitations" from this title and move the whole section to the other ALTER TABLE stuff further below